airflow/config_templates/default_airflow.cfg

```diff
@@ -1313,6 +1313,9 @@ worker_pods_queued_check_interval = 60
 # You may want this higher if you have a very large cluster and/or use ``multi_namespace_mode``.
 worker_pods_pending_timeout_batch_size = 100
 
+# The number of lines (from the end) of the logs to read from the worker pod's base container.
+read_log_lines = 100
```

Comment:
maybe `running_pod_tail_lines`?

because it seems pretty rare that you'd want to customize this, just to avoid another config param i'm tempted to suggest using a class constant that you could monkey patch in airflow local settings... but... i don't feel strongly about it.

Reply:
Thanks @dstandish . Renamed as suggested.

I fully understand the concern about adding another config param, but I still find this necessary. `100` lines seems reasonable for most cases, but when user want to have more flexibility, e.g. debugging their jobs, more log lines will be needed (this happens to me often in real works).

Comparing with different methods, like the one you shared above (class constant and monkey patch in local settings), I still find adding it as a config param is better.

Please let me know if these clarify, or should you have any other concern. Thanks a lot!